.deploy/helpers/deployOrder.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-console */
+import yaml from 'js-yaml';
+import fs from 'fs';
+import jsonpath from 'jsonpath';
+
+import { CLOUDFORMATION_SCHEMA } from 'js-yaml-cloudformation-schema';
+
+/**
+ * Based on export and imports in serverless templates, sort the list to get exports before any import.
+ * @param {array} files - List of paths to services files.
+ * @return {array} - Sorted list with exports first.
+ */
+export default files => {
+  const services = [];
+  // Collect all import and exports in the serverless files.
+  files.forEach(file => {
```

Comment:
Would use the js map utility function here to iterate over the files array
const services = files.map( file => { ...try catch logic } )